content/docs/command-reference/fetch.md

```diff
@@ -106,6 +107,12 @@ The `dvc remote` used is determined in order, based on
   workspace. This downloads tracked data for the entire commit history of the
   project.
 
+- `--max-size` - fetch data files/directories that are each below specified size
+  (bytes).
+
+-- `--type` - fetch data files/directories that are of a particular type
```

Comment:
what types exactly are supported?

Reply:
It's in the next line

> (metrics, plots).

Reply:
let's make a bit more explicit (it was not clear, e.g. that it doesn't support `data`): "`metrics` and `plots` are supported type values".

Reply:
Adjusted.